src/main/java/baritone/behavior/PathingBehavior.java

```diff
@@ -456,6 +460,11 @@ public BetterBlockPos pathStart() { // TODO move to a helper or util class
         return feet;
     }
 
+    @Override
+    public void onSettingChanged(SettingChangedEvent event) {
+        this.precomputedData.refresh();
```

Comment:
I think we'd want this to be more like `this.precomputedData = refreshed()`. Because we don't want to modify the precomputed data being used by the pathing thread, I'd worry about concurrent modification exception.

Reply:
Ok, actually all that's happening is modifying a boolean array, which can't crash due to concurrent modification, but still it's best practice to consistently separate objects modified on main game tick thread from objects used on pathing thread.